src/components/LHNOptionsList/OptionRowLHN.js

```diff
@@ -106,19 +115,8 @@ function OptionRowLHN(props) {
     const hasBrickError = optionItem.brickRoadIndicator === CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR;
     const defaultSubscriptSize = optionItem.isExpenseRequest ? CONST.AVATAR_SIZE.SMALL_NORMAL : CONST.AVATAR_SIZE.DEFAULT;
     const shouldShowGreenDotIndicator =
-        !hasBrickError &&
-        (optionItem.isUnreadWithMention ||
-            ReportUtils.isWaitingForIOUActionFromCurrentUser(optionItem) ||
-            (optionItem.isTaskReport && optionItem.isTaskAssignee && !optionItem.isCompletedTaskReport && !optionItem.isArchivedRoom));
+        !hasBrickError && (optionItem.isUnreadWithMention || optionItem.isWaitingForTaskCompleteFromAssignee || ReportUtils.isWaitingForIOUActionFromCurrentUser(optionItem));
```

Comment:
I did not introduced these changes! Is this normal on a merge workflow? This doesn't happen in rebase workflows.

Reply:
Ah, I imagine something got messes up with the re-push. I'd recommend just moving your changes to a new branch fresh from `main`

Reply:
😞 a lot of learnings, I tell you. Here is the new branch: https://github.com/redpanda-bit/Expensify-App/tree/fix/popover-showing.

Reply:
At this point, I think you can just create a PR for that branch and we'll close this one

Reply:
Here is the new PR: https://github.com/Expensify/App/pull/28614.